plugins/lib/plugin.ts

```diff
@@ -80,19 +86,37 @@ type SendTransactionResult = {
    * @returns The transaction response.
    */
   wait: (options?: TransactionWaitOptions) => Promise<TransactionResponse>;
-}
+};
 
 type GetTransactionRequest = {
   transactionId: string;
-}
+};
+
+/**
+ * Sign transaction request for Stellar.
+ */
+export type SignTransactionRequest = {
+  unsignedXdr: string;
```

Comment:
should field name be more generic in case some other network get to use that method?